src/xrt/drivers/illixr/illixr_component.cpp

```diff
@@ -75,15 +78,43 @@ extern "C" struct xrt_pose illixr_read_pose() {
 	return ret;
 }
 
-extern "C" void illixr_write_frame(unsigned int left,
-								   unsigned int right) {
+extern "C" void illixr_publish_gl_image_handle(GLuint handle, int num_images, int swapchain_index) {
```

Comment:
Use the prototype from the header file: `void illixr_publish_gl_image_handle(GLuint handle, uint32_t num_images, uint32_t swapchain_index);`